pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -799,7 +799,7 @@ type Status struct {
 // ReplicatedSpec represents the spec for replication in a pool
 type ReplicatedSpec struct {
 	// Size - Number of copies per object in a replicated storage pool, including the object itself (required for replicated pool type)
-	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Minimum=0
```

Comment:
I think the original is a good validation to have. It seems better to me to allow the `ReplicatedSpec` and `ErasureCodedSpec` to be pointers and nullable.

Reply:
This is restoring the behavior we had in 1.5 and previous schema. Changing to a pointer would involve a lot of code change. Sometimes there is the ideal and then what is practical. :)